util/etag.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package util
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"io"
+	"os"
+)
+
+// ComputeEtag returns etag for a file at path
+func ComputeEtag(path string) (string, error) {
```

Comment:
Yeah the etag is for requesting from s3 with If-None-Match head which is to prevent the updater from redownloading ~50 megs over and over in case of the updater having trouble... we might want to disable this feature to simplify the updater though